server/common/src/main/java/org/eclipse/lsp/cobol/common/copybook/CopybookName.java

```diff
@@ -29,22 +29,22 @@
 @AllArgsConstructor
 @Builder
 public class CopybookName {
-  String displayName;
-  String dialectType;
+  String name;
+  String dialect;
   String extension;
   public String getQualifiedName() {
```

Comment:
It would be either getName or getQualifiedName because there is no longer displayName attribute. getQualifiedName is selected because by inspecting some copybook related classes that they are used without just "name" that's why there was "displayName" instead of just "name".